presto-main/src/test/java/com/facebook/presto/operator/TestTableWriterOperator.java

```diff
@@ -246,6 +254,23 @@ public void testStatisticsAggregation()
         assertThat(info.getStatisticsCpuTime().getValue(NANOSECONDS)).isGreaterThan(0);
     }
 
+    private static Page getOutputPage(List<Type> types, int rowCount, int statsFieldCount, boolean lastPage)
+    {
+        PageBuilder pageBuilder = new PageBuilder(types);
+
+        pageBuilder.declarePosition();
+        BIGINT.writeLong(pageBuilder.getBlockBuilder(0), rowCount);
+        pageBuilder.getBlockBuilder(1).appendNull();
+        VARBINARY.writeSlice(
+                pageBuilder.getBlockBuilder(2),
+                wrappedBuffer(OPERATOR_EXECUTION_CONTEXT_CODEC.toJsonBytes(new OperatorExecutionContext(Lifespan.taskWide(), 0, 0, CommitGranularity.TABLE, lastPage))));
```

Comment:
nit: I would simply extract the `OPERATOR_EXECUTION_CONTEXT_CODEC.toJsonBytes(new OperatorExecutionContext(Lifespan.taskWide(), 0, 0, CommitGranularity.TABLE, lastPage)))` into a local variable and inline the `getOutputPage`

Reply:
Thanks for pointing this out! This is anyway not required because now we won't add a new position for output page, and we could still use `rowPagesBuilder`.